integration_test/third_party_apps_data/applications/mariadb/metadata.yaml

```diff
@@ -149,6 +149,7 @@ expected_metrics:
     unavailable_on:
       - ubuntu-2310-amd64
       - ubuntu-2310-arm64
+      - debian-12-arm64
```

Comment:
Why is `mariadb` being skipped on `debian-12-arm64`? It runs on `debian-12` (non-`arm64`), and since `mysql` is also skipped this means we have no MySQL/MariaDB coverage.

Reply:
@quentinmit just this one metric seems to be an issue for debian-12-arm64 (and I should only be skipping the one metric and not the entire test). 

From what I understood, there was a known discussion about this metric being problematic at one point so I assumed this wouldn't be a problem to skip. Let me know if this rings any bells or if you want to further discuss.

Reply:
Ah, the diff hunk was misleading and I thought it was talking about all the metrics, not just one. Line 44 is actually skipping MariaDB entirely on `debian-12`, with a different reason. If the tests otherwise pass aside from this one metric now, LGTM.